app/src/main/java/com/github/orkest/View/EditProfileActivity.kt

```diff
@@ -79,57 +79,68 @@ fun editProfileSetting(content: @Composable () -> Unit) {
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun EditProfileScreen() {
-    val notifSettingsItem = MenuItem(id = "notificationSettings", title = "Notifications", icon = Icons.Default.Notifications)
-    val privacyItem = MenuItem(id = "privacySettings", title = "Privacy", icon = Icons.Default.Phone)
-    val helpItem = MenuItem(id = "help", title = "Help", icon = Icons.Default.Info)
-
-    val items = listOf(notifSettingsItem, privacyItem, helpItem)
 
     val scaffoldState = rememberScaffoldState()
     val coroutineScope = rememberCoroutineScope()
 
 
-    Column(modifier = Modifier.fillMaxHeight()){
-        topBar()
-        IconButton(
-            onClick = { coroutineScope.launch {
-                if(scaffoldState.drawerState.currentValue == DrawerValue.Closed)
-                    scaffoldState.drawerState.open()
-                else
-                    scaffoldState.drawerState.close()
+    Scaffold(
+        // keep track of the state of the scaffold (whether it is opened or closed)
+        scaffoldState = scaffoldState,
+        topBar = { TopBar(coroutineScope = coroutineScope, scaffoldState = scaffoldState) },
+        // The content displayed inside the drawer when you click on the hamburger menu button
+        drawerContent = { CreateMenuDrawer() },
+
+        content = { padding ->
+            Modifier
+                .fillMaxHeight()
+                .padding(padding)
+            Column() {
+                // profile pic and edit button
+                EditProfileImage()
+                Divider()
+                MainBody()
+            }
+        },
+        drawerGesturesEnabled = true
+    )
+
```

Comment:
Do you think we'll have to modify this Scaffold object, because these objects can have maintenance issues and this would imply thorough testing, but just curious